storage/range.go

```diff
@@ -1962,5 +1955,7 @@ func (r *Range) ChangeReplicas(changeType proto.ReplicaChangeType, replica proto
 	if err != nil {
 		return util.Errorf("change replicas of %d failed: %s", desc.RaftID, err)
 	}
+	// Enqueue this range again to see if there are more changes to be made.
+	r.rm.ReplicateQueue().MaybeAdd(r, r.rm.Clock().Now())
```

Comment:
I'm not convinced this is the right place to do this. Feels like it would be better to do it in the queue itself. See storage/split_queue.go:process().